main.js

```diff
@@ -1428,7 +1428,10 @@ function installLibraries(callback) {
         for (let lib = 0; lib < libraries.length; lib++) {
             if (libraries[lib] && libraries[lib].trim()) {
                 libraries[lib] = libraries[lib].trim();
-                const libName = libraries[lib].split('@')[0];
+                let libName = libraries[lib];
+                if (libName.indexOf('@', 1) > -1) {
```

Comment:
Could you please adjust to prevent to execute the indexOf twice?

Reply:
Done